client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -403,6 +417,9 @@ const PatternAssembler: Step = ( { navigation, flow, stepName } ) => {
 
 	const stepContent = (
 		<div className="pattern-assembler__wrapper" ref={ wrapperRef } tabIndex={ -1 }>
+			{ isEnabled( 'pattern-assembler/notices' ) && (
```

Comment:
I haven't used an AsyncLoad component here because the file size is small and we have to import the function `getNoticeContent` from the same file. See `import Notices, { getNoticeContent } from './notices/notices';`.

I could move that function to another file, but it makes more sense to me to keep it all together.